wip_pacta_data_qa.Rmd

```diff
@@ -0,0 +1,545 @@
+
+1. List of fund ISINs and their Values impact on the overall PF value, filtered by direct_holding == FALSE and asset_type == "Funds"
+2. List of fund_isins that have ISIN == "MissingValue" and are therefore omitted by the first table. This, however, seems to come from the normalise_fund_data() function and may not be necessary. It is introduced on purpose for fund holdings, whose overall sum weight is smaller than 1. 
+
+**Actions:**
+
+* Check if the funds with higher impact are available in the fund_data file.
+* If they are not, try to get the corresponding fund data from LIM and add the data to the fund_data table before the next run.
+
+
+```{r}
+funds_coverage <- result_data_qa %>% 
+  filter(direct_holding == F &
+           asset_type == "Funds" &
+           investor_name != "Meta Investor") %>% 
+  select(fund_isin, currency,#isin, company_name,
+         value_usd, market_value) %>% 
+  group_by(fund_isin, currency) %>% 
+  summarise(sum_value_usd = round(sum(value_usd, na.rm=T)),
+            sum_market_value = round(sum(market_value, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(
+    share_missing_value = round(sum_value_usd/
+      (investors_overview %>% filter(investor_name=="Meta Investor") %>% pull(sum_value_usd)), 5)
+    ) %>% 
+  arrange(fund_isin)
+datatable(funds_coverage)
+
+funds_coverage_missing_value <- result_data_qa %>% 
+  filter(isin == "MissingValue" &
+           investor_name != "Meta Investor") %>% 
+  select(fund_isin, currency,#isin, company_name,
+         value_usd, market_value) %>% 
+  group_by(fund_isin, currency) %>% 
+  summarise(sum_value_usd = round(sum(value_usd, na.rm=T)),
+            sum_market_value = round(sum(market_value, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(
+    share_missing_value = round(sum_value_usd/
+      (investors_overview %>% filter(investor_name=="Meta Investor") %>% pull(sum_value_usd)), 5)
+    ) %>%  
+  arrange(fund_isin)
+datatable(funds_coverage_missing_value)
+
+
+```
+
+
+## Value by asset type
+**OPEN:**
+
+Target and Actions unclear
+One idea seems to be to check to share of sovereign bonds...
+
+```{r}
+value_by_asset_type <- result_data_qa %>% 
+  filter(investor_name != "Meta Investor") %>% 
+  group_by(asset_type, security_type) %>% 
+  summarise(sum_value_USD = round(sum(value_usd, na.rm = T))) %>% 
+  ungroup()
+datatable(value_by_asset_type)
+```
+
+## Port Weight check
+This table checks if the portfolio weights of the data inputs and outputs correspond.
+We expect that this is the case, i.e. the difference in port_weight should be negligible.
+
+1. Check if equity portfolio inputs match the outputs in terms of weighted value per company
+2. Check if corporate bond portfolio inputs match the outputs in terms of weighted value per company
+
+**Actions:**
+
+* If this is not the case, dig into what could have lead to changes of portfolio weights during the analysis.
```

Comment:
What are the potential reasons here for this to be different 😕. I think this has caused a bit of confusion in the past, as sometimes differences are acceptable, but it's been flagged. So we should possibly question this one again.

Reply:
Yea, I found that surprising as well. Let's see what the discussion on Monday brings